Resources/config/view.xml

```diff
@@ -12,6 +12,7 @@
         <parameter key="fos_rest.view_handler.default.class">FOS\RestBundle\View\ViewHandler</parameter>
         <parameter key="fos_rest.view_handler.jsonp.class">FOS\RestBundle\View\JsonpHandler</parameter>
         <parameter key="fos_rest.serializer.exception_wrapper_serialize_handler.class">FOS\RestBundle\Serializer\ExceptionWrapperSerializeHandler</parameter>
+        <parameter key="fos_rest.serializer.exception_wrapper_normalizer.class">FOS\RestBundle\Serializer\ExceptionWrapperNormalizer</parameter>
```

Comment:
I think we should remove this parameter. There are better ways for third-parties to overwrite/modify service definitions. Even Symfony itself doesn't introduce class name parameters for new services anymore (removing the parameter wouldn't be a BC break right now since there was no release since the PR has been merged).